.github/workflows/build-package.yml.template

```diff
@@ -109,16 +112,30 @@ jobs:
       - name: Tag release
         run: |
           (
+            set -euxo pipefail
             cd trigger
-            git tag $CRATE-$VERSION-$TARGET_ARCH
-            git push origin $CRATE-$VERSION-$TARGET_ARCH
+            git tag -f $CRATE-$VERSION $CRATE-$VERSION-$TARGET_ARCH
+            git push origin --tags
           )
 
-      - name: Releasing assets
-        uses: softprops/action-gh-release@v1
+      - name: Releasing assets in new release format
+        uses: svenstaro/upload-release-action@v2
         with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag_name: $CRATE-$VERSION
+          release_name: $CRATE-$VERSION
+          body: build $CRATE@$VERSION
+          file: |
+            ${{github.workspace}}/built.d/$CRATE-$VERSION-$TARGET_ARCH.tar.gz
+          overwrite: true
```

Comment:
This makes me uncomfortable. Could we put an `if` around this somehow, to skip it if the asset already exists?

Reply:
I've removed `overwrite` and also updates step "Tag release" to append to `$GITHUB_ENV` if the old or new tag is already present to skip these steps.